### PR TITLE
Add four new category types for new firmware to use

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-util-common.c [hunk 1]
```diff
@@ -670,6 +670,24 @@ fu_util_release_get_name (FwupdRelease *release)
 			 * at system bootup */
 			return g_strdup_printf (_("%s CPU Microcode Update"), name);
 		}
+		if (g_strcmp0 (cat, "X-Battery") == 0) {
+			/* TRANSLATORS: battery refers to the system laptop power source */
```
Comment (superm1) [inline src/fu-util-common.c:674]: I think that many things have BMUs, they don't have to be laptops :)

File: src/fu-util-common.c [hunk 2]
```diff
@@ -670,6 +670,24 @@ fu_util_release_get_name (FwupdRelease *release)
...
+			 * camera in the bezel or external USB webcam */
+			return g_strdup_printf (_("%s Camera Update"), name);
+		}
+		if (g_strcmp0 (cat, "X-TPM") == 0) {
+			/* TRANSLATORS: TPM refers to a Trusted Platform Module */
+			return g_strdup_printf (_("%s TPM Update"), name);
+		}
+		if (g_strcmp0 (cat, "X-Touchpad") == 0) {
+			/* TRANSLATORS: TouchPad refers to the flat pointing
+			 * device on a latop */
```
Comment (superm1) [inline src/fu-util-common.c:688]: again doesn't need to be a laptop here.

**Review by superm1 — APPROVED** [review]
just nits on the comments, no concerns.

File: src/fu-util-common.c (hunk 1 above)
Comment (hughsie) [reply src/fu-util-common.c:674]: Agreed, thanks.

File: src/fu-util-common.c (hunk 2 above)
Comment (hughsie) [reply src/fu-util-common.c:688]: Fixed.

**Pull request merged**